.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug
+about: Something does not work as expected
+----
+
+### Summary
+
+<!-- provide a summary here -->
```

Comment:
Somehow i dont see this in the markdown preview, is that just a `github` thing or do we need to push it in a code block?

Reply:
It's rendered as an HTML comment!